sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_schema_registry_client.py

```diff
@@ -211,6 +213,7 @@ def get_schema_properties(
                 :caption: Get schema id.
 
         """
+        format = get_case_insensitive_format(format)
```

Comment:
Is it necessary to change the case of format when building the request? or should it be enough to standardize the casing when performing the equality checking?

Reply:
This format is actually used in both the request and response checking, so I thought that I'd just standardize here so that I wouldn't have to do the check twice.